pkgs/by-name/bo/booklore/package.nix

```
@@ -0,0 +1,93 @@
+{
+  lib,
+  stdenvNoCC,
+  nixosTests,
+  fetchFromGitHub,
+  buildNpmPackage,
+  gradle_9,
+  makeWrapper,
+  openjdk25,
+}:
+let
+  version = "1.5.0";
```

```suggestion
  version = "1.10.0";
```

Update released very recently: https://github.com/booklore-app/booklore/releases/tag/v1.10.0

I've added the updated hashes as well.